_posts/2018-05-15-license-change.md

```diff
@@ -5,27 +5,27 @@ title: "Intent to adopt the Apache License 2.0 for Scala"
 by: Jorge Vicente Cantero and Adriaan Moors
 ---
 
-Good news everyone! By popular demand, we intend to change the Scala license
+Good news everyone! By popular demand, we intend to change the license that Scala is distributed under
 from the [3-clause BSD License](https://github.com/scala/scala/blob/2.13.x/LICENSE) to the [Apache License v2.0](https://www.apache.org/licenses/LICENSE-2.0.html).
 You have until 10 August 2018 to raise any objections (see below for contact info).
 
 # Is any of this legal advice?
```

Comment:
I'd move this to a footnote, it's kind of weird and formal as the first point, and I'd say redundant. It's important for lawyers to include these disclaimers in their notes to their clients, or a doctor to a patient, if they are just laying out the options, but I don't think there is any ambiguity here.